places-database-management-1/bulk-import-of-pois.md

```diff
@@ -0,0 +1,96 @@
+Places Import script
+=================
+A set of Python scripts have been created to simplify the batch import of POIs from a .csv file into a Places database by using the Places REST APIs. These scripts can be downloaded from this open source [git repo](https://github.com/adobe/places-scripts). Please check the [Organizational Requirements](../places-rest-apis/organizational-requirements.md) and [Adobe I/O integation](../places-rest-apis/adobe-i-o-integration/README.md) sections before running  these script to ensure you have access to the Places REST APIs.
+
+Following is information on the scripts. This information is also included in a readme file in the [git repo](https://github.com/adobe/places-scripts).
+
+## CSV file
+
+A sample .csv file, `places_sample.csv`, is part of this package and includes the required headers and a row of sample data. These headers are all lower case and correspond to the reserved metatdata keys that are used in the Places database. When you add headers, the additional columns are added to the Places database in a separate metadata section for each POI as key/value pairs.
+
+Here is a list of the columns and the values that you need to use:
+
+* lib_id
+
+  A valid library ID that is obtained from Places database.
```

Comment:
- "Places database" --> "POI database"
- Please add "the" between "from" and "POi database".